tests/integration/loss/test_functional.py

```diff
@@ -0,0 +1,60 @@
+import pytorch_testing_utils as ptu
+
+import torch
+from torch.nn.functional import mse_loss
+
+import pystiche.ops.functional as F
+
+
+class TestFunctionalLoss:
```

Comment:
Just seeing this: there is no need for this class here as the module name already implies that these are tests for the functionals. Plus, since it covers all test cases, this test case adds no value.

In order to keep the scope of the PR narrow, please revert this change and do a 1-to-1 port of the old file (minus my comment above). If there is something else here that needs to be addressed, we can do that in follow.up PRs.